src/components/FindWallet/WalletPersonasSidebar.tsx

```diff
@@ -0,0 +1,478 @@
+        linux: false,
+        windows: false,
+        macOS: false,
+        firefox: true,
+        chromium: true,
+        hardware: false,
+        open_source: true,
+        non_custodial: true,
+        hardware_support: false,
+        walletconnect: true,
+        rpc_importing: true,
+        nft_support: true,
+        connect_to_dapps: true,
+        staking: false,
+        swaps: false,
+        layer_2: true,
+        gas_fee_customization: true,
+        ens_support: true,
+        erc_20_support: true,
+        buy_crypto: false,
+        withdraw_crypto: false,
+        multisig: false,
+        social_recovery: false,
+      },
+    },
+    {
+      title: "I use multiple chains",
+      description:
+        "You use wallets on other chains and wants to get in ethereum",
+      featureHighlight: [
+        filterLabels.layer_2,
+        filterLabels.rpc_importing,
+        filterLabels.erc_20_support,
+        filterLabels.nft_support,
+      ],
+      presetFilters: {
+        android: true,
+        ios: true,
+        linux: false,
+        windows: false,
+        macOS: false,
+        firefox: true,
+        chromium: true,
+        hardware: false,
+        open_source: true,
+        non_custodial: true,
+        hardware_support: true,
+        walletconnect: true,
+        rpc_importing: true,
+        nft_support: true,
+        connect_to_dapps: true,
+        staking: false,
+        swaps: false,
+        layer_2: true,
+        gas_fee_customization: true,
+        ens_support: true,
+        erc_20_support: true,
+        buy_crypto: false,
+        withdraw_crypto: false,
+        multisig: false,
+        social_recovery: false,
+      },
+    },
+    {
+      title: "I'm all about NFT's",
+      description:
+        "You are someone that is all about NFTs a wallet has to ready",
+      featureHighlight: [
+        filterLabels.nft_support,
+        filterLabels.layer_2,
+        filterLabels.walletconnect,
+        filterLabels.hardware_support,
+      ],
+      presetFilters: {
+        android: true,
+        ios: true,
+        linux: false,
+        windows: false,
+        macOS: false,
+        firefox: true,
+        chromium: true,
+        hardware: false,
+        open_source: true,
+        non_custodial: true,
+        hardware_support: true,
+        walletconnect: true,
+        rpc_importing: false,
+        nft_support: true,
+        connect_to_dapps: true,
+        staking: false,
+        swaps: false,
+        layer_2: true,
+        gas_fee_customization: true,
+        ens_support: false,
+        erc_20_support: false,
+        buy_crypto: false,
+        withdraw_crypto: false,
+        multisig: false,
+        social_recovery: false,
+      },
+    },
+  ]
+
+  return (
+    <Container>
+      {personas.map((persona, idx) => {
+        return (
+          <Persona
+            selected={selectedPersona === idx}
+            onClick={() => {
+              setSelectedPersona(idx)
```

Comment:
This doesn't allow a user to deselect an active persona - we should support that.

Currently the only way to deselect a persona once it's been selected it to click on a different one.

Reply:
> This doesn't allow a user to deselect an active persona - we should support that.
> 
> Currently the only way to deselect a persona once it's been selected it to click on a different one.

Yes. it was a design bug. I had it in the first drafts and somehow disappear form the final proposal. 
![Screen Shot 2022-06-17 04 45 06 PM](https://user-images.githubusercontent.com/1120748/174332537-96285ca2-a41b-419e-b2db-540ed631415c.png)

The idea is something like this